src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -6572,7 +6572,7 @@ private TupleTypeSyntax ParseTupleType()
                 var close = this.EatToken(SyntaxKind.CloseParenToken);
                 var result = _syntaxFactory.TupleType(open, list, close);
 
-                if (!result.ContainsDiagnostics && list.Count < 2)
+                if (list.Count < 2)
```

Comment:
>if (list.Count < 2) [](start = 16, length = 19)

It looks like a change here isn't covered by the new unit-tests. Is it possible to add one?